lib/updatehub/Kconfig

```diff
@@ -0,0 +1,88 @@
+#
+# Copyright (c) 2018, 2019 O.S.Systems
+#
+# SPDX -License-Identifier: Apache-2.0
+#
+
+menuconfig UPDATEHUB
+	bool"UpdateHub Firmware Over-the-Air support"
+	select FLASH
+	select REBOOT
+	select IMG_MANAGER
+	select BOOTLOADER_MCUBOOT
+	select MPU_ALLOW_FLASH_WRITE
+	select NETWORKING
+	select NEWLIB_LIBC
+	select NET_UDP
+	select NET_SOCKETS
+	select NET_SOCKETS_POSIX_NAMES
+	select COAP
+	select NET_CONFIG_NEED_IPV4
+	select NET_CONFIG_SETTINGS
+	select DNS_RESOLVER
+	select JSON_LIBRARY
+	select TINYCRYPT
+	select TINYCRYPT_SHA256
+	select HWINFO
+	help
+	  UpdateHub is an enterprise-grade solution which makes simple to
+	  remotely update all your embbeded devices in the field. It
+	  handles all aspects related to sending Firmware Over-the-Air
+	  (FOTA) updates with maximum security and efficiency, while you
+	  focus in adding value to your product.
+
+config UPDATEHUB_POLL_INTERVAL
+	int "Time to poll interval (in minutes)"
+	default 1440
+	range 0 43200
+	depends on UPDATEHUB
+	help
+	  Set the interval that the UpdateHub update server will be polled.
+	  This time interval is zero and 43200 minutes(30 days).
+
+config UPDATEHUB_PRODUCT_UID
```

Comment:
Can this use the hwinfo (drivers/hwinfo) for UID?

Reply:
No, because product-uid is about products on the UpdateHub Cloud, is not related to hardware.